apps/subscription/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from apps.user.models import User
+
+# Create your models here.
+
+
+# Subject Data Model
+class Subject(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
+# SubjectUsers is a bridge between subject and user
+class SubjectUsers(models.Model):
+    project = models.ForeignKey(
```

Comment:
I am thinking this table alllows users to save the subjects they are interested in? Great idea. Question, why is "project" the name of the foreign key to a subject? Wouldn't it be "subject"?

Reply:
project was used temporarily but i forgot to change it to subject, will fix